plugins-compat-tester-model/src/main/java/org/jenkins/tools/test/model/PluginCompatTesterConfig.java

```diff
@@ -99,6 +99,11 @@
     private String gaeSecurityToken;
     // GoogleAppEngin base url for plugin compat tester
     private String gaeBaseUrl;
+    
+    //Allows Jenkins test version specification.
+    //Use the Jenkins test classes from the same parent version of this plugin or the most recent release of Jenkins
+    //true= same version; false= most recent version
+    private boolean testVersion = false;
```

Comment:
I was following the convention in the rest of the file.  There are a lot of multiline comments and descriptions that would probably be suited for Javadoc.